website/versioned_docs/version-2.0.0/core-plugins-simple-inventory.md

```diff
@@ -0,0 +1,65 @@
+
+**Option 2** If your external inventory system produces periodic exports but cannot be queried in real time, then you may want to keep the included `simple-inventory` plugin and create a script that reads the exported files and updates the `SimpleInventory` collection as necessary. If you do this in a custom plugin, then it's best to call `context.mutations.updateSimpleInventory` with `context.isInternalCall` set to `true`, and let that take care of properly updating the collection. If you can't do it in a custom plugin, then your external script should be written to look similar to the `updateSimpleInventory` mutation function in the `simple-inventory` plugin.
+
+Example of option 2:
+
+```
+const STANDARD_LOW_INVENTORY_THRESHOLD = 10;
+
+async function externalSystemInventoryUpdateWorker(context, productId, productVariantId, shopId, inventoryInStock) {
+  await context.mutations.updateSimpleInventory(
+    { ...context, isInternalCall: true },
+    {
+      productConfiguration: {
+        productId,
+        productVariantId
+      },
+      shopId,
+      canBackorder: false,
+      lowInventoryWarningThreshold: STANDARD_LOW_INVENTORY_THRESHOLD,
+      isEnabled: true,
+      inventoryInStock
+    },
+    { returnUpdatedDoc: false } // improves performance
+  );
+}
+```
+
+## How to configure products with no tracked inventory
+
+There are a few reasons why some products may not have tracked inventory. Some may be digital or conceptual products that you can make unlimited copies of. You may build some products to specification after they are ordered. Or you may drop ship some products and have no insight into what quantity are available.
+
+These are all situations in which you likely want to have no known inventory quantity but yet allow ordering any quantity. In other cases, though, you may be synchronizing inventory data from an external system, and a variant with no tracked inventory might mean it hasn't synced yet and therefore should be considered sold out (and ideally also invisible).
+
+To accommodate either situation, the core "inventory" plugin has a shop setting, found on the Shop Settings page of the operator UI. The setting, `canSellVariantWithoutInventory`, determines whether a variant with no inventory information can be sold. This setting is `true` by default.
+- If you leave it `true`, then you will need to enable inventory tracking, disable back-ordering, and set "in stock" inventory to `0` for each variant that you want to appear sold out.
+- If you change it to `false`, then for products that should allow unlimited sales, you have two options:
+    - Enable inventory tracking, set "in stock" inventory to `0` and enable back-ordering, or
```

Comment:
Inconsistant use of hyphens in the word `back-ordering`. I'm not sure which is correct, but we should stick to one.